lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/body.html

```diff
@@ -26,20 +26,21 @@
             {% include "goal_reminder_banner.html" %}
             <div class="goal-reminder-body-wrapper" style="width:inherit; font-size: 14px; font-size: 0.875rem; padding: 0 20px 0 5%; margin: 36px 20px 0 17px; margin: 2.25rem 1.25rem 0 1.0625rem;">
                 <h3 style="
-                    font-size: 22px;
-                    font-size: 1.38rem;
+                    font-size: 25px;
```

Comment:
increase font size

Reply:
Why is font-size specified twice? Is that like, a kind of comment, about where the weird rem value came from?

Reply:
no, many email clients don't support rem, so then if you want to set the font size you're supposed to put a px fallback (otherwise it will just set the font size to something random if only rem is specified)

Reply:
The web was a mistake.

Reply:
yeah we should go back to snail mail